packages/webpack/template-webpack-plugin/src/WebEncodePlugin.ts

```diff
@@ -87,7 +87,13 @@ export class WebEncodePlugin {
               styleInfo: encodeOptions['styleInfo'],
               manifest: encodeOptions.manifest,
               cardType: encodeOptions['cardType'],
-              pageConfig: encodeOptions.compilerOptions,
+              pageConfig: {
```

Comment:
We already have a `pageConfig` field on `encodeOptions`, which is equal to `encodeData.compilerOptions`:

https://github.com/lynx-family/lynx-stack/blob/1a65d8779cf13be892051653a3634c6ec26099c1/packages/webpack/template-webpack-plugin/src/WebEncodePlugin.ts#L76

I think you should merge the two configs there (in `beforeEncode`), instead of in `encode`.